README.md

```diff
@@ -1,29 +1,127 @@
-# Convos - Multiuser chat application
+![Convos](http://convos.by/images/logo.png)
 
+Convos is the simplest way to use IRC. It is always online, and accessible to
+your web browser, both on desktop and mobile. Run in on your home server, or
+cloud service easily. It can be deployed to Docker-based cloud services, or
+you can just run it as a normal Mojolicious application, using any of the
+Deployment Guides.
 
- Convos is a multiuser chat application built with Mojolicious.
+![Screenshot](http://convos.by/images/screenshot.png)
```

Comment:
We need a new screenshot... Maybe we can use something from https://drive.google.com/open?id=0B_tCHIu9z7YSOTY5WnE5Mzk3M0E ?